docs/release_notes/pending_release.rst

```diff
@@ -4,7 +4,16 @@ Pending Release Notes
 Updates / New Features
 ----------------------
 
-* Add scikit-learn SVM classifier
+CI
+
+* Add workflow to inherit the smqtk-core publish workflow.
+
+Miscellaneous
+
+* Add scikit-learn SVM classifier.
```

Comment:
This should be under an "algorithms" section. I will do this update and approve.